demos/projects/ESPRESSIF/adu/README.md

```diff
@@ -35,6 +35,11 @@ This sample will allow you to update an ESP32 over the air (OTA) using Azure Dev
     For other Operating Systems or to update an existing installation, follow [Espressif official documentation](https://docs.espressif.com/projects/esp-idf/en/latest/esp32/get-started/index.html#get-started).
 
 1. [Powershell](https://docs.microsoft.com/powershell/scripting/install/installing-powershell?view=powershell-7.2)
+
+1. Azure CLI and Azure IoT Module
```

Comment:
Do we need changes to the H7/L4+ instructions as well?

Reply:
Yes but I unfortunately did not update that documentation for those :/

I'll make a card to make the changes and then we can more or less copy the L475 ones to the other ST boards.